numpy/_core/_add_newdocs.py

```diff
@@ -6945,9 +6945,10 @@ def refer_to_array_attribute(attr, method=True):
     array([False, True, False])
 
     >>> np.array([1.2, object(), "hello world"],
-    ...          dtype=StringDType(coerce=True))
-    ValueError: StringDType only allows string data when string coercion
-    is disabled.
+    ...          dtype=StringDType(coerce=False))
+    Traceback (most recent call last):
+        ...
+    ValueError: StringDType only allows string data when string coercion is disabled.
```

Comment:
Each time I read this, I parse the "only" the opposite way initially, is it just me?  Anyway, nitpick unrelated to this.